### ISSUE TITLE
Inaccurate constrainedBy

### ISSUE THREAD

**Issue by damienpicard** [open-ideas/IDEAS#522]
Some partial models do not have correct `constrainedby`.
For example, `IDEAS.Buildings.Components.Interfaces.PartialOpaqueSurface` (from which the walls are extended) have the following constrainedby:
```
replaceable IDEAS.Buildings.Data.Constructions.CavityWall constructionType
    constrainedby Data.Constructions.CavityWall(final insulationType=
        insulationType, final insulationTickness=insulationThickness)
    "Type of building construction"
```

This is confusing as the user's walls will not extend from `CavityWall` but rather from `IDEAS.Buildings.Data.Interfaces.Construction` and therefore the scroll down menu for the parameter `constructionType` will not include their walls type.

**Comment by Mathadon** [comment]
This will be fixed in #518

**Comment by Mathadon** [comment]
Closed in #518
